tools/format.ts

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env -S deno run --quiet --allow-read --allow-write --allow-run --allow-env
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { main } from "./check.ts";
+
+await main("format", Deno.args[0]);
```

Comment:
Maybe these could be `format` and `lint` and passing proper strings could be done internally in `tools/check.ts`?. Not a big deal though

Reply:
I'll punt on this for now but I probably have a few random fixes for these tools and I can do it then.